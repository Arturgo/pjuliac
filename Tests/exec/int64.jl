
println(9223372036854775807)
println(-9223372036854775808)
println(9223372036854775807 + 1)
println(-9223372036854775808 - 1)
println(2 ^ 63)
println(2 ^ 63 - 1)
println(- 2 ^ 63)
println(1729 ^ 42)
