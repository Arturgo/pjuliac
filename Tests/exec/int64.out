9223372036854775807
-9223372036854775808
-9223372036854775808
9223372036854775807
-9223372036854775808
9223372036854775807
-9223372036854775808
5971570120393124737
